pkg/controller/elasticsearch/client/v7.go

```diff
@@ -42,7 +43,12 @@ func (c *clientV7) StartTrial(ctx context.Context) (StartTrialResponse, error) {
 }
 
 func (c *clientV7) AddVotingConfigExclusions(ctx context.Context, nodeNames []string) error {
-	path := fmt.Sprintf("/_cluster/voting_config_exclusions/%s", strings.Join(nodeNames, ","))
+	var path string
+	if c.version.IsSameOrAfter(version.From(7, 8, 0)) {
+		path = fmt.Sprintf("/_cluster/voting_config_exclusions?node_names=%s", strings.Join(nodeNames, ","))
+	} else {
+		path = fmt.Sprintf("/_cluster/voting_config_exclusions/%s", strings.Join(nodeNames, ","))
```

Comment:
If `c.version` is empty (client constructed from `client.NewElasticsearchClient()`), we end up in this `else` path, which is fine.
Maybe worth explicitly pointing out in a comment though?

Reply:
Not sure I follow your argument wrt to `client.NewElasticsearchClient()` but I added a comment.

Reply:
I meant client which do not have a version set, which I think is the case when built from `client.NewElasticsearchClient()`.
I'm 👍 with the comment.